src/connector/src/common.rs

```diff
@@ -41,7 +41,7 @@ use crate::source::nats::source::NatsOffset;
 // The file describes the common abstractions for each connector and can be used in both source and
 // sink.
 
-pub const BROKER_REWRITE_MAP_KEY: &str = "broker.rewrite.endpoints";
```

Comment:
won't the renaming causing backward compatibility issue?
& why we need to change the key?

Reply:
Oh, didn't realize that. I revert it back.
Previously the field cannot be deserialized correctly, so I try to change the name and it works, but I found there is a mistake.